nodes/curves/Curves.py

```diff
@@ -18,6 +18,17 @@ def IsBezierCurve(blenderObject):
         if spline.type != 'BEZIER': return False
 
     return True
+    
+deltaParameter = 0.001
+def ParameterIsZero(parameter):
+    if parameter < deltaParameter: return True
+    
+    return False
+
+def ParameterIsOne(parameter):
+    if parameter + deltaParameter > 1.0: return True
```

Comment:
Done and done. 
I don't think I should create a new pull request for this, right? I did commit and sync my code.